src/TRestAxionXrayWindow.cxx

```diff
@@ -285,7 +285,7 @@ if (fWindowType == "stripped") {
 /// \brief It returns the vector element index, from `fEnergy`, that is just below the given input energy.
 ///
 Int_t TRestAxionXrayWindow::GetEnergyIndex(Double_t energy) {
```

Comment:
```suggestion
Int_t TRestAxionXrayWindow::GetEnergyIndex(Double_t energy) const {
```

(the header definition should also be updated).

Reply:
ok, I pushed the update also to the header